setup.py

```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env python
-
-from distutils.core import setup
+from setuptools import setup, find_packages
 
 setup(
     name='rtmbot',
-    version='0.2.0',
+    version='0.2.11',
     description='A Slack bot written in python that connects via the RTM API.',
-    author='Ryan Huber',
-    author_email='rhuber@gmail.com',
+    author='Ryan Huber and Jeff Ammons',
+    author_email='jeff@jeffammons.net',
     url='https://github.com/slackhq/python-rtmbot',
-    packages=['rtmbot'],
+    packages=find_packages(),
+    entry_points={'console_scripts': ['rtmbot=rtmbot.bin.run_rtmbot:main']},
+    install_requires=[
+        'python-daemon',
```

Comment:
Sounds good to me. I think it was partially implemented previously and I removed it.